components/favourites/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from 'react';
+import { Tag } from '#components-ui/tag';
+import { formatIntFr } from '#utils/helpers';
+import { useLocalStorage } from 'hooks';
+
+export default function Favourites() {
+  const [shortCuts, setShortCuts] = useState([]);
+  const [recentVisits] = useLocalStorage('favourites-siren', []);
+
+  useEffect(() => {
+    setShortCuts(recentVisits);
+  }, [recentVisits]);
+
+  return (
+    shortCuts.length > 0 && (
```

Comment:
Why not use directly `rectenVisits` ?

Reply:
It creates hydration conflict as recent visit is empty on server rendered page. I wanted to ask you if you know a more idiomatic way to solve this

Reply:
Et oui, c'est logique :+1: 
Et non, pas de meilleure idée. A checker si ça ne plombe pas trop le score de CLS sur pagespeed insight.
La solution alternative serait de mettre les dernières visites dans la session pour avoir le prérendu côté serveur, mais ça aussi ses inconvénients.

Reply:
Extremement bien vu cette question du CLS. Ou alors on fixe la hauteur ? Je tente un dev en ce sens